core/src/display_object/edit_text.rs

```diff
@@ -1129,6 +1132,42 @@ impl<'gc> EditText<'gc> {
             }
         }
     }
+
+    fn initialize_as_broadcaster(&self, activation: &mut Activation<'_, 'gc, '_>) {
+        let write = self.0.write(activation.context.gc_context);
+        let object = write.object.unwrap();
```

Comment:
Let's avoid the unwrap here, and explicitly check that there is an Object:
`let object = if let Some(object) = write.object { .. } else { return; }`
Because eventually there will be no AVM1 object if this is an AVM2 SWF. (Currently EditText is kind of behind the times and constructs an AVM1 object even if the movie is AVM2 -- this will be cleaned up eventually).